### PR TITLE
Add dumpling chinese manual

### PR DESCRIPTION
**Pull request by 3pointer**

### CODE REVIEW

**Comment by GregoryIan** [comment]
add link into readme.md

File: docs/cn/user-guide.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Dumpling 使用手册
+
+[Dumpling](https://github.com/pingcap/dumpling) 是针对 MySQL/TiDB 数据库的 dump 成 SQL 文件工具。
```
Comment (GregoryIan) [inline docs/cn/user-guide.md:3]:
```suggestion
[Dumpling](https://github.com/pingcap/dumpling) 是支持以 SQL 文本或者 CSV 格式将 MySQL/TiDB 数据库的数据的导出工具。
```

File: docs/cn/user-guide.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+其它基本参数 -u -p -P 用法跟 Mydumper 保持一致。
+
+更多具体用法可以使用 -h, --help 进行查看。
+
+## Mydumper 相关参考
+
+[Mydumper usage](https://github.com/maxbube/mydumper/blob/master/docs/mydumper_usage.rst)
+
+[TiDB Mydumper 使用文档](https://pingcap.com/docs-cn/stable/reference/tools/mydumper/)
```
Comment (lichunzhu) [inline docs/cn/user-guide.md:29]: How about linking download url to the end?

File: docs/cn/user-guide.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+[Dumpling](https://github.com/pingcap/dumpling) 是针对 MySQL/TiDB 数据库的 dump 成 SQL 文件工具。
+
+设计初衷是为了替代 [Mydumper](https://github.com/pingcap/mydumper), 所以基本用法可以参考 Mydumper, 
+当然在实现中没有完全照搬 Mydumper, 因此存在与 Mydumper 不同的用法。
+
+下表罗列了一些与 Mydumper 不同之处。
+
+| 主要参数 |     |
+| --------| --- |
+| -B 或 --database| 导出数据库名称|
```
Comment (GregoryIan) [inline docs/cn/user-guide.md:12]: same with `mydumper`?

Comment (lichunzhu) [inline docs/cn/user-guide.md:12]: I think we should emphasize `-H` index here because it's quit different from mydumper.

**Pull request merged**